README.md

```diff
@@ -5,50 +5,66 @@ PCP PMDA for StatsD in C
 
 You need to have **chan** [repo](https://github.com/tylertreat/chan) and **HdrHistogram_c** [repo](https://github.com/HdrHistogram/HdrHistogram_c) installed in your /usr/local dir. You also need to have **Ragel** installed. 
 
```

Comment:
Can you add the diagram to the readme? Just in a simple text form, add a paragraph about the design of the project. Something like:

```
LISTENER -> PARSER -> AGGREGATOR - XXXX - YYYY
```

Or use some tool like https://textik.com if you want.

Reply:
Added textik.com chart in d7ce1c8, but it was broken due to the way default white-space collapsing works in HTML, I don't think I can / should style it via inline css. d890387 made them simpler in plain text.